package.json

```diff
@@ -11,38 +11,38 @@
     "preview": "vite preview"
   },
   "dependencies": {
-    "@mantine/core": "^7.5.0",
-    "@mantine/hooks": "^7.5.0",
+    "@mantine/core": "^7.5.3",
+    "@mantine/hooks": "^7.5.3",
     "@mantinex/mantine-logo": "^1.0.1",
-    "@tabler/icons-react": "^2.46.0",
+    "@tabler/icons-react": "^2.47.0",
     "axios": "^1.6.7",
     "json2typescript": "^1.5.1",
     "loader-utils": "3.2.1",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-icons": "^5.0.1",
-    "react-router-dom": "^6.22.0",
+    "react-router-dom": "^6.22.1",
     "react-scripts": "5.0.1",
-    "styled-components": "5.3.9",
+    "styled-components": "6.1.8",
     "zustand": "^4.5.1"
   },
   "devDependencies": {
     "@reactivers/hooks": "^1.0.15",
     "@types/js-cookie": "^3.0.6",
-    "@types/react": "^18.2.43",
-    "@types/react-dom": "^18.2.17",
-    "@typescript-eslint/eslint-plugin": "^6.14.0",
-    "@typescript-eslint/parser": "^6.14.0",
+    "@types/react": "^18.2.58",
+    "@types/react-dom": "^18.2.19",
+    "@typescript-eslint/eslint-plugin": "^7.0.2",
+    "@typescript-eslint/parser": "^7.0.2",
     "@vitejs/plugin-react": "^4.2.1",
-    "eslint": "^8.55.0",
+    "eslint": "^8.57.0",
     "eslint-plugin-react-hooks": "^4.6.0",
     "eslint-plugin-react-refresh": "^0.4.5",
     "js-cookie": "^3.0.5",
     "jwt-decode": "^4.0.0",
-    "postcss": "^8.4.33",
-    "postcss-preset-mantine": "^1.12.3",
+    "postcss": "^8.4.35",
+    "postcss-preset-mantine": "^1.13.0",
     "postcss-simple-vars": "^7.0.1",
-    "typescript": "4.9.5",
-    "vite": "^5.0.8"
+    "typescript": "5.3.3",
+    "vite": "^5.1.4"
   }
```

Comment:
Is it possible to check that switching from version 4.9.5 to version 5.3.3 does not create conlfit in the docker environment (hot reload)?

Reply:
it's ok